src/js/UI/userInterface.js

```diff
@@ -0,0 +1,83 @@
+import { $, $$ } from '../util/general.js';
+import storage from '../storage/storage.js';
+
+const template = {
+  skeletonUI: `<li class="skeleton">
+  <div class="image"></div>
+  <p class="line"></p>
+  <p class="line"></p>
+</li>`,
```

Comment:
인덴트 맞춰주시는 겟 좋을 것 같습니다!

```suggestion
   skeletonUI: `
  <li class="skeleton">
    <div class="image"></div>
    <p class="line"></p>
    <p class="line"></p>
  </li>`,
```

Reply:
수정 하였습니다!